.gitignore

```diff
@@ -10,4 +10,10 @@ bin
 out
 obj
 build
-TestResults
\ No newline at end of file
+TestResults
+
+# Include ONLY launch/task files in .vscode
```

Comment:
@gfmatthews This may be closer to what we want.  This will ignore everything in `.vscode` EXCEPT launch.json and task.json.

You are right, we do want to be able to share changes to how the app runs. I think this strikes the right balance